src/Helper/ConfigurationHelper.php

```diff
@@ -214,7 +219,33 @@ protected function loadFromSpecificFile($filePath)
         }
 
         $this->detectedPath = $filePath;
-        return include $filePath;
+
+        try {
```

Comment:
No need for a try/catch here, can just return directly. Good to add the `@throws` though!